src/helpers/deleteArrayItem.js

```diff
@@ -1,5 +1,6 @@
-export const deleteArrayItem = (items, moveFromIndex) => {
-  items.splice(moveFromIndex, 1);
+export const deleteArrayItem = (mainArray, moveFromIndex) => {
+  const array = mainArray;
+  array.splice(moveFromIndex, 1);
 
-  return items;
+  return array;
```

Comment:
this still mutates the original array:

![Bildschirmfoto 2022-05-20 um 13 20 35](https://user-images.githubusercontent.com/1942953/169517420-02c72c9c-096d-4bb8-b21c-407f2d35b496.png)

Reply:
[096984c](https://github.com/ikuseiGmbH/smart-village-app-app/pull/536/commits/096984c64d992d9da3140016b63ccff108960dcf)